common/app/views/fragments/bashers/ThisIsTheNHS.scala.html

```diff
@@ -0,0 +1,41 @@
+@()(implicit request: RequestHeader)
+
+<div class="gs-container">
+    <div class="this-is-the-nhs">
+        <h1 class="title title--first">this is</h1>
```

Comment:
A classical BEM approach would have us replace `this-is-the-nhs title` with `this-is-the-nhs__title` (or perhaps `nhs-campaign__title` for ease of typing). Off the top of my head, I can think of a few advantages to doing things this way:

1. It is clear that we are using a campaign-specific style rather than a generic `title` widget. It is also clear that an `nhs-campaign__grid` class is particular to the component, rather than a site-wide layout helper.
2. There is no chance of namespace collision with other generic styles, like a non-campaign `title` class.
3. We are keeping specificity low by avoiding a descendant combinator that we do not actually need.
4. As is, there is an invisible coupling between the div and the H1. It is not _explicit_ that the class on the container also styles the children. BEM makes the dependencies between containers and children very obvious.

What do you think?